docs/testing.md

```diff
@@ -42,6 +42,7 @@ Then, simply run (from the narrative root directory) `make test`.
 This calls a few subcommands, and those can be run independently for specific uses:
 
 - `make test-frontend-unit` will run only the unit tests on the frontend (i.e. those with the Karma runner)
+- `make test-integration` will run the frontend integration tests that make use of webdriver.io to simulate the browser on a locally instantiated Narrative, but running against live KBase services.
```

Comment:
Do you want to add in `make test-frontend`, which runs the integration and frontend unit tests?

Reply:
Yup, I'll add that to the docs.

Reply:
Added.